packages/config-utils/proxy.js

```diff
@@ -84,26 +149,13 @@ module.exports = ({
     routes = require(routesPath);
   }
 
-  if (routes) {
+  if (routes?.length > 0) {
```

Comment:
This expects the routes to be an array but by default, it is an object. This will disable any existing proxy configurations using the object.

Reply:
Reverted that.